### ISSUE TITLE
Hearing File attachment in SLACK? 

### ISSUE THREAD

**Issue by vipin733** [botman/botman#559]
- BotMan Studio  Version: 2.0
- PHP Version 7.0:
- Slack Messaging Service(s):
- SLack  Driver:

### Description:
I'm implementing hear fileattachment but its seems not working, I have fallback hear method and file received hear method like    

here is my botman.php 

```
$botman->receivesFiles(function($bot, $files) {
    foreach ($files as $file) {
        $url = $file->getUrl();
        $payload = $file->getPayload();
    }
    $bot->reply('recived');
});  // file receiving 

$botman->fallback(function($bot) {
    $bot->reply('Sorry I'm buggy now!');
});   //fallback 

```

now every time when I try to upload image its reply "Sorry I'm buggy now instead of received 

what is solution please give some guidance I trying solve this since 4 days but could not found solution 

Thanks

**Comment by charliebois** [comment]
Slack driver does not support files, the slackRTM driver does. See here https://botman.io/2.0/driver-slack#realtime-api

**Comment by vipin733** [comment]
I switch to RTM but still its not working

**Comment by mpociot** [comment]
Fixed in Slack Driver 1.1.1
